.buildkite/scripts/run_job.py

```diff
@@ -0,0 +1,129 @@
+#!/usr/bin/env python3
+# -*- encoding: utf-8
+
+import argparse
+import os
+import subprocess
+import sys
+
+import git_utils as git
+
+from sbt_dependency_tree import Repository
+
+
+def check_call(cmd):
+    """
+    A wrapped version of subprocess.check_call() that doesn't print a
+    traceback if the command errors.
+    """
+    print("*** Running %r" % " ".join(cmd))
+    try:
+        return subprocess.check_call(cmd)
+    except subprocess.CalledProcessError as err:
+        print(err)
+        sys.exit(err.returncode)
+
+
+def make(*args):
+    """Run a Make command, and check it completes successfully."""
+    check_call(["make"] + list(args))
+
+
+def should_run_sbt_project(repo, project_name, changed_paths):
+    project = repo.get_project(project_name)
+
+    interesting_paths = [p for p in changed_paths if not p.startswith(".sbt_metadata")]
+
+    if ".travis.yml" in interesting_paths:
```

Comment:
Is this still going to work? I suppose it would, but maybe worth moving over to another config file?

Reply:
No. It's not going to work, it should reference `.buildkite/pipeline.yml`.